cmd/openshift-install/create.go

```diff
@@ -323,6 +326,15 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 		},
 	)
 
+	// If the CVO failed, print out the last failing operator statuses
+	if err != nil && clusterVersion != nil {
+		for _, c := range clusterVersion.Status.Conditions {
+			if c.Type == configv1.OperatorFailing {
+				logrus.Warn(c.Message)
```

Comment:
We know we errored out, so why `WARN`, just returns that error...?

Reply:
Fixed :arrow_down: